app/scripts/extract_scheduled_gtfs.py

```diff
@@ -146,70 +147,109 @@ def ingest_transit_stations(
             station_id=row["stop_id"],
             station_name=row["stop_name"],
             location=Point(row["stop_lat"], row["stop_lon"], srid=4326),
-            mode=transit_mode,
+            mode=assign_mode(row, url),
         )
         print(
             f'Observation created: {row["city"]}, {row["stop_id"]}, '
             f'{row["stop_name"]}, Point({row["stop_lat"]},{row["stop_lon"]})'
             f", mode {obs.mode}"
         )
-        print(obs)
         try:
             obs.save()
-            print(f"Observation {i} saved to PostGIS")
+            print(f"Observation {i} saved to PostGIS\n")
         except Exception as e:  # TODO: make this more specific
             print(e)
             print("Skipping import of this observation\n")
     print("Ingestion complete")
 
 
-def handroll_geometrize_routes(city: str, feed) -> gpd.GeoDataFrame:
-    """Associate each route and its properties with an appropriate GEOS LineString.
-    Handrolls a simplified version of the gtfs_kit library's geometrize_routes()
-    method, which broke on some transit feeds with which we tried it.
-    Results in one shape per route, irrespective of direction.
-
-    Warning: Some users may not be able to test this function in any way
-    other than running this script, due to the 'ImproperlyConfigured' error
-    in which attempts to import types from django.contrib.gis.geos fail to find
-    the path for GDAL.
-
-    TODO: write test that establishes all needed fields are there for every city
-    """
-    geom_shapes = feed.geometrize_shapes()
-    geom_shapes.loc[:, "city"] = city
+def assign_mode(row, url: str) -> int:
+    """Take in data about a transit stop and assign it the proper mode per
+    GTFS standards (which isn't in the stops.txt). Is not super scalable
+    if more cities are added.
+
+    TODO: Think about how to redesign for easier scaling or perhaps redo with
+    some creative joins"""
+
+    LIGHT_RAIL = 0
+    SUBWAY = 1
+    RAIL = 2
+    BUS = 3
+    AERIAL_LIFT = 6
+
+    if url == METRA_URL:
+        return RAIL
+    elif url in BUS_URLS:
+        return BUS
+    elif url == MTA_SUBWAY_URL:
+        return SUBWAY
+    elif url == CTA_URL:
+        # every CTA stop_id is an integer. Bus stop_ids currently end at 18,709
+        # El stop_ids currently start at 30,000
+        # This may fail
+        if int(row["stop_id"]) > 30000:
+            return SUBWAY
+        else:
+            return BUS
+    elif url == TRIMET_URL:
+        if "MAX" in row["stop_name"]:
+            return LIGHT_RAIL
+        elif "WES" in row["stop_name"]:
+            return RAIL
+        elif "Tram Terminal" in row["stop_name"]:
+            return AERIAL_LIFT  # there's an aerial tram!!
+        else:
+            return BUS
+
+
+def handroll_multiline_routes(city: str, feed) -> gpd.GeoDataFrame:
```

Comment:
I get this is fine for now, but I don't find the name of the function particularly clear on what should do.